### ISSUE TITLE
After updating the patches in office 2016, 'displayDialogAsync' method started giving the error - Unable to get property displayDialogAsync of undefined or null reference

### ISSUE THREAD

**Issue by mnkumar8** [OfficeDev/office-js#1748]
I got this error before and it got resolved by updating with office 2016 patch. I started seeing this error again after updating office 2016 with recent updates. What could be the reason for this error?
I confirm that office.js is loaded and can able to read excel data from Office.initialize block.
The supported excel API version on my machine is 1.10.  
Please see the attached picture for the office version that I am using.
![VamshiOfficeVersion](https://user-images.githubusercontent.com/11781512/112447188-0639ad80-8d77-11eb-82d7-80d3e86a36c9.JPG)
  

## Expected Behavior

'displayDialogAsync' should show the dialog box for the office version I have mentioned above.

## Current Behavior

I am seeing error - Unable to get property displayDialogAsync of undefined or null reference.

## Steps to Reproduce, or Live Example

* Link to live example: ______

* Additional details:
  1. ______
  2. ______
  3. ______

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: ______
* Host [Excel, Word, PowerPoint, etc.]: ______
* Office version number: ______
* Operating System: ______
* Browser (if using Office on the web): ______

## Useful logs

- [ ] Console errors
- [ ] Screenshots
- [ ] Test file (if only happens on a particular file)

**Comment by ElizabethSamuel-MSFT** [comment]
@mnkumar8 Thanks for letting us know about this.

@lumine2008 Can you take a look, or assign to an appropriate person to investigate?

Thanks.

**Comment by lumine2008** [comment]
This is related to UX, so let me loop @lliu113. thanks.

**Comment by lliu113** [comment]
@mnkumar8 - sorry you're hitting this! Are other APIs working, or is it just the dialog API that is throwing this error? Can you provide a snippet of the code you're using to display the dialog?

**Comment by mnkumar8** [comment]
Excel JS API seems to be working fine. I saw the error for dialog API. This error got fixed after installing the office patch 'osfclient-x-none'.

Below is the code I am using to display the dialog box.
            Office.context.ui.displayDialogAsync(window.location.origin + "/SelectedItemsPage.html", { height: 50, width: 23 },
                function (asyncResult) {
                    xl.dialog = asyncResult.value;
                    xl.dialog.addEventHandler(Office.EventType.DialogEventReceived, xl.handleDialogEvent); 
//xl.handleDialogEvent is a js method to handle dialog box events/errors
                }
            );

**Comment by lliu113** [comment]
@mnkumar8 - just want to confirm that you followed the instructions for the dialog as specified in the [Dialog API requirement sets article](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/dialog-api-requirement-sets)? Osfclient-x-none is the one you want - it seems like you've downloaded it but just wanted to confirm.

**Comment by mnkumar8** [comment]
Yes, i have followed the instructions in the specified link and downloaded it and it resolved the issue now. I have seen the error 'Unable to get property displayDialogAsync of undefined or null reference' twice on a particular machine. 
1) First time, it got fixed by updating the office 2016 by our IT team. 
2) After few days, the same error appeared again. 
3) I have then created this ticket to know the cause of this error. It was already been resolved by our IT team and then the error appeared again.
4) Second time, it got resolved by installing this specific patch 'Osfclient-x-none' after following the instructions in the link (link specified in the above comment).
